modules/flowable-engine/src/test/java/org/flowable/engine/test/bpmn/mail/EmailSendTaskTest.java

```diff
@@ -146,6 +146,54 @@ public void testCcAndBccWithForceTo() throws Exception {
             );
     }
 
+    @Test
+    @Deployment
+    public void testOnlyBccAddress() throws Exception {
+        runtimeService.startProcessInstanceByKey("onlyBccAddress");
+
+        List<WiserMessage> messages = wiser.getMessages();
+        assertEmailSend(messages.get(0), false, "Hello world", "This is the content", "flowable@localhost", Collections.emptyList(),
+            Collections.emptyList());
+
+        assertThat(messages)
+            .extracting(WiserMessage::getEnvelopeSender, WiserMessage::getEnvelopeReceiver)
+            .containsExactlyInAnyOrder(
+                tuple("flowable@localhost", "mispiggy@activiti.org")
+            );
+    }
+
+    @Test
+    @Deployment
+    public void testOnlyCcAddress() throws Exception {
+        runtimeService.startProcessInstanceByKey("onlyCcAddress");
+
+        List<WiserMessage> messages = wiser.getMessages();
+        assertEmailSend(messages.get(0), false, "Hello world", "This is the content", "flowable@localhost", Collections.emptyList(),
+                Collections.emptyList());
+
+        assertThat(messages)
+                .extracting(WiserMessage::getEnvelopeSender, WiserMessage::getEnvelopeReceiver)
+                .containsExactlyInAnyOrder(
+                        tuple("flowable@localhost", "mispiggy@activiti.org")
+                );
+    }
+
+    @Test
+    @Deployment
+    public void testOnlyToAddress() throws Exception {
+        runtimeService.startProcessInstanceByKey("onlyToAddress");
+
+        List<WiserMessage> messages = wiser.getMessages();
+        assertEmailSend(messages.get(0), false, "Hello world", "This is the content", "flowable@localhost", Collections.emptyList(),
+                Collections.emptyList());
```

Comment:
`assertEmailSend` can verify the `to` address specifically. Lets do:

```suggestion
        assertEmailSend(messages.get(0), false, "Hello world", "This is the content", "flowable@localhost", "mispiggy@activiti.org",
                Collections.emptyList());
```

Reply:
Done